spec/controllers/redirections_controller_spec.rb

```diff
@@ -1,5 +1,19 @@
 require "rails_helper"
 
+RSpec.shared_examples_for "disallowing a similar URL" do |action, item|
```

Comment:
How would you feel about putting this logic inline in each test? It's only two tests, and I think it's easier to understand what's required to set up the test. If you still feel like the length is hurting readability, maybe just a method would be a good middle-ground instead of this RSpec-only feature?

I've never used shared examples and it took me some time to understand what was happening. That frustration is surely coloring my opinion here. See also [my recent blog post](https://thoughtbot.com/blog/the-self-contained-test) on keeping specs all in one place.

Reply:
Absolutely. Now that there are fewer controller tests, I'll remove this block.